frontend/components/shared/Footer.tsx

```diff
@@ -0,0 +1,205 @@
+import { Image, Link } from '@nextui-org/react'
+
+const contactMethods = [
```

Comment:
[nits]

If a variable is fixed, and wont change dynamically in the application and is outside the component scope, its better to use all caps like:

CONTACT_METHODS. However, I think that we might need the CONTACT_METHODS variable in the future for more important stuff, so its better to use something like COMPANY_CONTACT_METHODS